sdk/communication/communication-chat/swagger/README.md

```diff
@@ -11,14 +11,16 @@ description: Chat Client
 generate-metadata: false
 license-header: MICROSOFT_MIT_NO_VERSION
 output-folder: ../src/generated
-input-file: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master/specification/communication/data-plane/Microsoft.CommunicationServicesChat/stable/2021-03-07/communicationserviceschat.json
+tag: package-chat-2021-03-07
+require: https://raw.githubusercontent.com/Azure/azure-rest-api-specs/bf081421869ccd31d9fd87084b07a1e246aee310/specification/communication/data-plane/Microsoft.CommunicationServicesChat/readme.md
 model-date-time-as-string: false
 optional-response-headers: true
 use-extension:
   "@autorest/typescript": "6.0.0-dev.20210114.1"
   "@autorest/modelerfour": "4.15.442"
 azure-arm: false
 add-credentials: false
+disable-async-iterators: true
```

Comment:
@knvsl Thanks for helping me out. We need to add this to the readme. Kim can provide the history for why this is needed

Reply:
@LuChen-Microsoft can confirm this is case - if I recall right, without the flag list methods are generated as private and this was causing issues trying to convert the types from generated models to the SDK models?

Reply:
This is currently set in project package.json file. We need to remove it from package script.